### PR TITLE
Add an entry in glossary for GVR

### PR DESCRIPTION
**Pull request by Richabanker**

### CODE REVIEW

File: content/en/docs/reference/glossary/group-version-resource.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Group Version Resource
+id: gvr
+date: 2023-07-24
+full_link: /docs/concepts/architecture/unknown-version-interoperability-proxy#Interoperability-proxying
+short_description: >
+  The group, version and resource of a Kubernetes API resource. 
+
+aka:
```
Comment (sftim) [inline content/en/docs/reference/glossary/group-version-resource.md:9]:
```suggestion
aka: ["GVR"]
```

File: content/en/docs/reference/glossary/group-version-resource.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: Group Version Resource
+id: gvr
+date: 2023-07-24
+full_link: /docs/concepts/architecture/unknown-version-interoperability-proxy#Interoperability-proxying
+short_description: >
+  The group, version and resource of a Kubernetes API resource.
```
Comment (sftim) [inline content/en/docs/reference/glossary/group-version-resource.md:7]:
```suggestion
  The API group, API version and name of a Kubernetes API. 
```

File: content/en/docs/reference/glossary/group-version-resource.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+aka:
+tags:
+- architecture
+---
+Means of representing unique Kubernetes API resource.
+
+<!--more-->
+
+GVRs are the group, version, resource of an API resource that are used to define and distinguish different Kubernetes objects.
```
Comment (sftim) [inline content/en/docs/reference/glossary/group-version-resource.md:17]:
```suggestion
Group, Version, Resources (GVRs) specify the API group, API version, and resource (name for the object kind as it appears in the URI) associated with accessing a particular id of object in Kubernetes.
GVRs let you define and distinguish different Kubernetes objects, and to specify a way of accessing
objects that is stable even as APIs change.
```

File: content/en/docs/reference/glossary/group-version-resource.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+title: Group Version Resource
+id: gvr
+date: 2023-07-24
+full_link: /docs/concepts/architecture/unknown-version-interoperability-proxy#Interoperability-proxying
```
Comment (sftim) [inline content/en/docs/reference/glossary/group-version-resource.md:5]:
```suggestion
```

File: content/en/docs/reference/glossary/group-version-resource.md
Comment (sftim) [inline content/en/docs/reference/glossary/group-version-resource.md:1]: At the HTTP level, `/api/v1/namespaces/default/pods/hello-world-1` is a resource. So is `/api/v1/namespaces/default/pods` (it's a collection of Pods), but I don't think it's quite what people think of when they write GVK or GVR.

I don't have a good answer here. Something to watch out for though.

---

Resources also mean the cgroup kind: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/

File: content/en/docs/reference/glossary/group-version-resource.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+title: Group Version Resource
```
Comment (sftim) [inline content/en/docs/reference/glossary/group-version-resource.md:2]:
```suggestion
title: Group, Version, Resource
```

**Review by sftim — COMMENTED** [review]
Thanks.

/lgtm
(yes, we could merge it)

**Comment by Richabanker** [comment]
@sftim could you take a look this PR and let know if it looks ok to be merged?

File: content/en/docs/reference/glossary/group-version-resource.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+aka: ["GVR"]
+tags:
+- architecture
+---
+Means of representing unique Kubernetes API resource.
+
+<!--more-->
+
+Group, Version, Resources (GVRs) specify the API group, API version, and resource (name for the object kind as it appears in the URI) associated with accessing a particular id of object in Kubernetes.
```
Comment (kbhawkey) [inline content/en/docs/reference/glossary/group-version-resource.md:16]: nit: The `id` is `gvr`, should the title use `Group Version Resource`?
nit: Group Version Resource (GVRs) specify the API group, version and resource ...

File: content/en/docs/reference/glossary/group-version-resource.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
+---
+title: Group, Version, Resource
```
Comment (kbhawkey) [inline content/en/docs/reference/glossary/group-version-resource.md:2]: Group Version Resource

File: content/en/docs/reference/glossary/group-version-resource.md (hunk 6 above)
Comment (Richabanker) [reply content/en/docs/reference/glossary/group-version-resource.md]: Done.

File: content/en/docs/reference/glossary/group-version-resource.md (hunk 7 above)
Comment (Richabanker) [reply content/en/docs/reference/glossary/group-version-resource.md]: Done.

**Comment by tengqm** [comment]
Seeing several green lights before, kicking this in.
/lgtm
/approve

**Pull request merged**